ingest/bin/post_process_metadata.py

```diff
@@ -0,0 +1,91 @@
+#! /usr/bin/env python3
+
+"""Reformat pandas DataTables for a pathogen build.
+
+Expecting one argument, a NCBI Virus metadata.tsv file
+"""
+# ===== Dependencies
+import argparse
+import os
+import sys
+
+import numpy as np
+import pandas as pd
+
+
+def parse_args():
+  # Main help command
+  parser = argparse.ArgumentParser(
+    description = "USAGE: python post_process_metadata.py --metadata 'data/metadata.tsv'"
+  )
+  # Add first argument
+  parser.add_argument(
+    "--metadata",
+    help = "NCBI Virus metadata.tsv file.",
+    required = True
+  )
+  parser.add_argument(
+    "--outfile", default="processed_metadata.tsv",
+    help = "Processed metdata file name. [default: processed_metadata.tsv] ",
+    required = False
+  )
+  
+  return parser.parse_args()
+
+# === Private methods
```

Comment:
There's a lot of variability in how folks use the leading underscore to indicate private functions throughout Nextstrain projects, but since these functions are technically not private and often appear in utility scripts like this instead of published Python libraries, I think we tend to skip this convention.

Reply:
Ah I could probably convert them to private functions. The name is a signal to later-me that the function is not supposed to be used on its own, a warning that the original code was written for a specific context (e.g. "requires certain columns to exist in a record") and is not generalized yet. 

Open to discussion though, I won't die on this hill.

Reply:
Yeah, it's a very minor aesthetic point and using the underscore definitely conveys the "written for a specific context" and use-at-your-own-risk background, so it makes sense here. I only pointed it out to note that Python doesn't have a way to enforce private functions like other languages do, so the naming is a convention.